package introduction.ExceptionHandling;

public class Demo {
    public static void main(String[] args) throws Exception {
        Main.divide(3, 4);
    }
}
